articles/private-link/disable-private-endpoint-network-policy.md

```diff
@@ -71,25 +71,23 @@ $vnet | Set-AzVirtualNetwork
 This section describes how to disable subnet private endpoint policies using Azure CLI. Use [az network vnet subnet update](/cli/azure/network/vnet/subnet#az_network_vnet_subnet_update) to disable the policy.
 
 ```azurecli
-az network vnet subnet update \ 
+az network vnet subnet update \
```

Comment:
Space (` `) is not allowed after `\`. The space breaks the [line continuation ](https://www.gnu.org/software/bash/manual/bash.html#Escape-Character-1) notation.